enterprise/frontend/src/embedding-sdk/hooks/private/use-init-data.ts

```diff
@@ -43,10 +53,8 @@ export const useInitData = ({ config }: InitDataLoaderParameters) => {
     dispatch(setLoginStatus({ status: "uninitialized" }));
```

Comment:
I just realized this is already the default state.
https://github.com/metabase/metabase/blob/effd25b065f75e3f982f27e1937c32798b33dd51/enterprise/frontend/src/embedding-sdk/store/reducer.ts#L55

Is there any point in deliberately setting it here again? This makes me think the status was something else before it is set to `uninitialized` which could be confusing.

Reply:
The idea was that if the user for whatever reason had a bad configuration that caused an error in a Metabase component, they might change the config. At that point, `useInitData` would be triggered (from `AppInitializeController`) and reset the `loginStatus` to `uninitialized` so it could attempt to reconnect

Reply:
I see you remove this in the latest change. But even if this was to be there, I'm not sure how it would set the login status back to `uninitialized` because the hook will only run once on `dispatch` change, which theoretically, is equivalent to just the first render since dispath never changes.

Reply:
ah yeah that's fair. I'll leave it as is then